src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/location/importer/LocationType.kt

```diff
@@ -9,24 +9,36 @@ import uk.gov.justice.digital.hmpps.pecs.jpc.location.Location
 private const val TYPE = 1
 private const val SITE = 2
 private const val AGENCY = 3
+private const val ACTIVE_INACTIVE = 6
 
-enum class LocationType(val index: Int, val label: String) {
+enum class LocationType(val label: String, private val activeFieldPosition: Int = ACTIVE_INACTIVE) {
 
-    COURT(5, "COURT"),
-    HOSPITAL(9, "HOSPITALS"),
-    IMMIGRATION(10, "IMMIGRATION"),
-    OTHER(12, "OTHER"),
-    POLICE(6, "POLICE"),
-    PRISON(8, "PRISON"),
-    STCSCH(11, "STC&SCH");
+    COURT("Courts", 8), // The position of this cell in the court tab column differs to the majority by two.
+    HOSPITAL("Hospitals"),
+    IMMIGRATION("Immigration"),
+    OTHER("Other"),
+    POLICE("Police", 7), // The position of this cell in the police tab column differs to the majority by one.
```

Comment:
For consistency / ease of reading, I would require all tabs to pass in the position of the active column (no default), then remove the comment